src/app-bar.jsx

```diff
@@ -314,26 +313,26 @@ const AppBar = React.createClass({
           {children}
       </Paper>
     );
-  },
+  }
 
+  @autobind
```

Comment:
I'm sorry but I completely disagree! Doing that makes the components extremely hard to maintain! I had one hell of bad time with dialog's doc page component. This is not a good pattern. and using arrow would make it look like this:

```js
_foo = () => {
  // ... 
};

```
It's not so bad, is it?